catalogue/webapp/utils/json-ld.ts

```diff
@@ -0,0 +1,33 @@
+import { Work as WorkType } from '@weco/common/model/catalogue';
+import { objToJsonLd } from '@weco/common/utils/json-ld';
+
+export function workLd(work: WorkType) {
```

Comment:
I've tried adding some, but the way we use these helpers is a little odd – we seem to be putting linked data into our models in places, which blows up because TypeScript thinks they're the wrong type.

I want to say "they're a WorkType and a JsonLd at the same time", but I don't know how to express that in the type system.